src/app/containers/Footer/index.jsx

```diff
@@ -1,7 +1,13 @@
+import styled from 'styled-components';
 import React from 'react';
 import SitewideLinks from '@bbc/psammead-sitewide-links';
 import BrandContainer from '../Brand';
 
+// 3.5rem = 56px = 7 GEL_SPACING's
```

Comment:
Why are we not using `GEL_SPACING` directly? Just curious - no need to change.

```suggestion
// 3.5rem = 56px = 7 x GEL_SPACING
```